File: app/src/main/java/com/example/adam/myapplication/ui/main/MainActivity.java
package com.example.adam.myapplication.ui.main;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.design.widget.BottomNavigationView;
import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentTransaction;
import android.support.v7.app.AppCompatActivity;
import android.view.MenuItem;

import com.example.adam.myapplication.R;

public class MainActivity extends AppCompatActivity implements MainContract.MainView {

    private MainContract.MainPresenter presenter;

    private BottomNavigationView bottomNavigation;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        getComponents();
        setPresenter();
    }

    private void getComponents() {
        bottomNavigation = findViewById(R.id.bottom_navigation);
        bottomNavigation.setSelectedItemId(0);
    }

    private void setPresenter() {
        this.presenter = new MainPresenter(this);
        setListeners();

        presenter.onViewAttached();
    }

    private void setListeners() {
        bottomNavigation.setOnNavigationItemSelectedListener(new BottomNavigationView.OnNavigationItemSelectedListener() {
            @Override
            public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
                menuItem.setChecked(true);

                switch(menuItem.getItemId()) {
                    case R.id.action_board:
                        presenter.onBoardSelected();
                        break;

                    case R.id.action_doctor:
                        presenter.onDoctorSelected();
                        break;

                    case R.id.action_pill:
                        presenter.onPillsSelected();
                        break;

                    case R.id.action_measurement:
                        presenter.onScoresSelected();
                        break;
                }
                return false;
            }
        });
    }

    @Override
    public void changeContentView(Fragment fragment) {
        FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
        transaction.replace(R.id.fragment_container, fragment);
        transaction.commit();
    }

    public void goToBoard() {
        bottomNavigation.setSelectedItemId(R.id.action_board);
    }
}

File: app/src/main/java/com/example/adam/myapplication/ui/main/MainActivity.kt
package com.example.adam.myapplication.ui.main

import android.os.Bundle
import android.support.v4.app.Fragment
import android.support.v7.app.AppCompatActivity
import com.example.adam.myapplication.R
import com.example.adam.myapplication.ui.board.BoardFragment
import com.example.adam.myapplication.ui.doctor.tabs.DoctorTabsFragment
import com.example.adam.myapplication.ui.drugs.drugs_tabs.DrugsTabsFragment
import com.example.adam.myapplication.ui.scores.scores_tabs.ScoresTabsFragment
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        setListeners()
        changeContentView(BoardFragment())
    }

    private fun setListeners() {
        setBottomNavigationListener()
    }

    private fun setBottomNavigationListener() {
        bottom_navigation.setOnNavigationItemSelectedListener { menuItem ->
            menuItem.isChecked = true

            when (menuItem.itemId) {
                R.id.action_board -> changeContentView(BoardFragment())

                R.id.action_doctor -> changeContentView(DoctorTabsFragment())

                R.id.action_pill -> changeContentView(DrugsTabsFragment())

                R.id.action_measurement -> changeContentView(ScoresTabsFragment())
            }
            false
        }
    }

    private fun changeContentView(fragment: Fragment) {
        val transaction = supportFragmentManager.beginTransaction()
        transaction.replace(R.id.fragment_container, fragment)
        transaction.commit()
    }

    fun goToBoard() {
        bottom_navigation.selectedItemId = R.id.action_board
        changeContentView(BoardFragment())
    }
}
